packages/connect/CHANGELOG.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "version": "2.0.0",
```

Comment:
should this be an RC version until the rest of the RC packages are out?

Reply:
@fabioberger thoughts here? Maybe it should be 2.0.0-rc.1?